src/frontend/components/component-tree/component-tree.ts

```diff
@@ -45,4 +45,7 @@ export class ComponentTree {
     }
   }
 
+  trackById(index: number, node: any): string {
```

Comment:
This PR looks good to me
if I could say something about it would be that the trackById function is duplicated in two different classes, however not sure if there is a common place to put it
either way this ^ comment is just cosmetic, so I'd say :shipit: